config/module.config.php

```diff
@@ -1544,6 +1544,7 @@
         ],
         'Laminas\I18n\Validator\IsInt' => [
             'locale' => 'string',
+            'strict' => 'bool',
```

Comment:
Unfamiliar with the test suite of this package - can any other maintainer of this package verify if this change can be testes?

Reply:
It's a configuration setting, for when you pass an array of options to the validator.

Honestly, we don't have any coverage for this configuration other than in the laminas-i18n, laminas-filter, laminas-validator, and laminas-inputfilter packages. I've verified that this option exists, however, so I'm good with adding it.